src/wp-includes/class-wp-image-editor.php

```diff
@@ -375,17 +374,18 @@ protected function get_output_format( $filename = null, $mime_type = null ) {
 		 * @see WP_Image_Editor::get_output_format()
 		 *
 		 * @since 5.8.0
+		 * @since 6.7.0 The default was changed from array() to array( 'image/heic' => 'image/jpeg' ).
 		 *
 		 * @param string[] $output_format {
 		 *     An array of mime type mappings. Maps a source mime type to a new
-		 *     destination mime type. Default empty array.
+		 *     destination mime type. Default maps uploaded HEIC images to JPEG output.
 		 *
 		 *     @type string ...$0 The new mime type.
 		 * }
 		 * @param string $filename  Path to the image.
 		 * @param string $mime_type The source image mime type.
 		 */
-		$output_format = apply_filters( 'image_editor_output_format', array(), $filename, $mime_type );
+		$output_format = apply_filters( 'image_editor_output_format', array( 'image/heic' => 'image/jpeg' ), $filename, $mime_type );
```

Comment:
For future maintainability, I'd suggest adding a default filter to do the mapping. This will prevent the need to update the default in three location if it changes in the future. It will allow plugin authors to use a simple `remove_filter()` call if they want to maintain HEIC as HEIC.

If you accept this, a `@see` annotation in the docblock would be lovely

Reply:
Plugin authors can `unset( $formats['image/heic'] )` using `image_editor_output_format` to maintain HEIC. I think that's a little clearer than unsetting a default filter.

Agree de-duplicating the default array would be nice for maintainability though. Maybe can add a wrapper function a la `wp_get_image_editor_output_format`?

Reply:
> Agree de-duplicating the default array would be nice for maintainability though. Maybe can add a wrapper function a la `wp_get_image_editor_output_format`?

Works for me.

Reply:
https://github.com/WordPress/wordpress-develop/pull/7034/commits/db78c272f7b06ba273c04afc8194b5b5351fa9de